releasenotes/notes/50132.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+issue:
+  - 48362
+releaseNotes:
+  - |
+    **Added** the annotation `gateway.istio.io/address-type` to allow `istio` class Gateways to use `ClusterIP` for status addresses.
+    **Added** the ability to annotate workloads or services with `istio.io/use-waypoint` pointing to Gateways of arbitrary gateway classes.
```

Comment:
Do we have a doc with all the details, including what annotations are supported, etc ? Would be best to link to the doc too.

Reply:
I hope this annotation doesn't live long. Maybe we don't mention it in the release note as if it's something we want to encourage.